sdk/appconfiguration/app-configuration/sample.env

```diff
@@ -1,3 +1,16 @@
 # This is the connection string for your app-configuration resource
 # You can get this from the Azure portal.
-AZ_CONFIG_CONNECTION=<app-configuration connection string goes here>
\ No newline at end of file
+AZ_CONFIG_CONNECTION=<app-configuration connection string goes here>
+AZ_CONFIG_ENDPOINT=<ex: https://sample.azconfig.io>
+
+# Register a new application in AAD and assign the "App Configuration Data Owner" role to it
```

Comment:
These docs are really wonderful.

Reply:
They're a slightly modified version of what we have in EventHubs :)